docs/02.API-REFERENCE.md

```diff
@@ -5393,20 +5391,31 @@ main (void)
 {
   jerry_init (JERRY_INIT_EMPTY);
 
-  static uint32_t save_literal_buffer[256];
+  static jerry_char_t literal_buffer[256];
+  static uint32_t snapshot_buffer[256];
   const jerry_char_t *code_for_literal_save_p = (const jerry_char_t *) "var obj = { a:'aa', bb:'Bb' }";
+  size_t code_for_literal_save_size = strlen ((const char *) code_for_literal_save_p);
+
+  jerry_value_t generate_result = jerry_generate_snapshot (NULL,
+                                                           0,
+                                                           code_for_literal_save_p,
+                                                           code_for_literal_save_size,
+                                                           0,
+                                                           snapshot_buffer,
+                                                           256);
+  size_t snapshot_size = (size_t) jerry_get_number_value (generate_result);
+  jerry_release_value (generate_result);
 
-  size_t literal_sizes = jerry_parse_and_save_literals (code_for_literal_save_p,
-                                                        strlen ((const char *) code_for_literal_save_p),
-                                                        false,
-                                                        save_literal_buffer,
-                                                        sizeof (save_literal_buffer) / sizeof (uint32_t),
-                                                        true);
+  const size_t literal_size = jerry_get_literals_from_snapshot (snapshot_buffer,
+                                                                snapshot_size,
+                                                                literal_buffer,
+                                                                256,
+                                                                true);
 
-  if (literal_sizes != 0)
+  if (literal_size != 0)
   {
-    FILE *literal_file_p = fopen ("literals.txt", "w");
-    fwrite (save_literal_buffer, sizeof (uint8_t), literal_sizes, literal_file_p);
+    FILE *literal_file_p = fopen ("literals.txt", "wb");
```

Comment:
note: `jerry_get_literals_from_snapshot` is called with `is_c_format=true`, so the .txt extension is a bit counterintuitive in the example. (this is nothing functional, just an observation)